arm-mediaservices/2015-10-01/swagger/media.json

```diff
@@ -0,0 +1,679 @@
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "description": "Name of the resource group within the Azure subscription.",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "mediaServiceName",
+            "description": "Name of the Media Service.",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "pattern": "^[a-z0-9]",
+            "minLength": 3,
+            "maxLength": 24
+          },
+          {
+            "name": "MediaService",
+            "description": "Media Service properties needed for update.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/MediaService"
+            }
+          }
+        ]
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Media/mediaservices/{mediaServiceName}/regenerateKey": {
+      "post": {
+        "summary": "Regenerate the key for a Media Service.",
+        "operationId": "MediaServices_RegenerateKey",
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/RegenerateKeyOutput"
+            }
+          },
+          "202": {
+            "description": "Accepted"
+          },
+          "default": {
+            "description": "Detailed error information.",
+            "schema": {
+              "$ref": "#/definitions/ApiError"
+            }
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "description": "Name of the resource group within the Azure subscription.",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "mediaServiceName",
+            "description": "Name of the Media Service.",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "pattern": "^[a-z0-9]",
+            "minLength": 3,
+            "maxLength": 24
+          },
+          {
+            "name": "RegenerateKeyInput",
+            "description": "Properties needed to regenerate the Media Service key.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/RegenerateKeyInput"
+            }
+          }
+        ]
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Media/mediaservices/{mediaServiceName}/listKeys": {
+      "post": {

```

Comment:
Post. Updating swagger.

From: John Hart [mailto:notifications@github.com]
Sent: Wednesday, June 29, 2016 2:08 PM
To: Azure/azure-rest-api-specs <azure-rest-api-specs@noreply.github.com>
Cc: Brian Blum <brianbl@microsoft.com>; Mention <mention@noreply.github.com>
Subject: Re: [Azure/azure-rest-api-specs] Media Services version 2015-10-01 (#376)


In arm-mediaservices/2015-10-01/swagger/media.json<https://github.com/Azure/azure-rest-api-specs/pull/376#discussion_r69027968>:

> +            "maxLength": 24

> +          },

> +          {

> +            "name": "RegenerateKeyInput",

> +            "description": "Properties needed to regenerate the Media Service key.",

> +            "in": "body",

> +            "required": true,

> +            "schema": {

> +              "$ref": "#/definitions/RegenerateKeyInput"

> +            }

> +          }

> +        ]

> +      }

> +    },

> +    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Media/mediaservices/{mediaServiceName}/listKeys": {

> +      "post": {

Is this a post or a get?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/Azure/azure-rest-api-specs/pull/376/files/5e6e26edeffb270c3e702bbd1756d46a7559d074#r69027968>, or mute the thread<https://github.com/notifications/unsubscribe/AEH9bD5R6u_6zArQI9y5dJyAW-YJ7gbVks5qQt6bgaJpZM4JAhvv>.